clang-tools-extra/clang-doc/tool/ClangDocMain.cpp

```diff
@@ -49,6 +49,7 @@
 using namespace clang::ast_matchers;
 using namespace clang::tooling;
 using namespace clang;
+static llvm::ExitOnError ExitOnErr;
```

Comment:
```suggestion

static llvm::ExitOnError ExitOnErr;
```
Nit: newline between using and the declaration

Reply:
actually, can you move this to be below all the cl::opts and constants? around line 131 seems like an OK spot, but I'll leave the precise location to you.

Reply:
Moved before the function definitions.